IntelliTect.TestTools.Console.Tests/ConsoleAssertTests.cs

```diff
@@ -51,6 +51,20 @@ public void ConsoleTester_HelloWorld_TrimLF()
             }, true);
         }
 
+        [TestMethod]
+        [DataRow(@"[49mMontoya", "Montoya")]
```

Comment:
Rather than putting in the unprintable escape character in the code, I would use the `\u001b` (or a constant) to be clear what that character is. There are lots of unprintable characters that will render as the box